pkgs/os-specific/linux/zfs/default.nix

```
@@ -227,17 +228,17 @@ in {
 
   zfsUnstable = common {
     # check the release notes for compatible kernels
-    kernelCompatible = kernel.kernelOlder "5.20";
-    latestCompatibleLinuxPackages = linuxPackages_5_19;
+    kernelCompatible = kernel.kernelOlder "6.1";
+    latestCompatibleLinuxPackages = linuxPackages_6_0;
 
     # this package should point to a version / git revision compatible with the latest kernel release
     # IMPORTANT: Always use a tagged release candidate or commits from the
     # zfs-<version>-staging branch, because this is tested by the OpenZFS
     # maintainers.
-    version = "2.1.6";
-    # rev = "0000000000000000000000000000000000000000";
+    version = "2.1.7";
```

Personally I don't mind, because `version` doesn't really have a meaning beyond the store path name, but some people prefer this
```suggestion
    version = "unstable-2021-10-21";
```
or maybe this might also be a good middle ground
```suggestion
    version = "2.1.7-staging-2021-10-21";
```

---

ah... that name does make sense